File: Advanced/PageAndRDLX/CustomTileProvider/VB.NET/TestDesignerPro/HelperForm.vb
Public Class HelperForm

	Private Sub rtfHelp_LinkClicked(sender As Object, e As LinkClickedEventArgs) Handles rtfHelp.LinkClicked
		Process.Start(New ProcessStartInfo With {
			.FileName = e.LinkText,
			.UseShellExecute = True
		})
	End Sub

	Private Sub HelperForm_FormClosing(sender As Object, e As FormClosingEventArgs) Handles MyBase.FormClosing
		e.Cancel = True
		Hide()
	End Sub

End Class

File: Advanced/PageAndRDLX/CustomTileProvider/C#/TestDesignerPro/HelperForm.cs
using System.Diagnostics;
using System.Windows.Forms;

namespace ActiveReports.Samples.TestDesignerPro
{
	public partial class HelperForm : Form
	{
		public HelperForm()
		{
			InitializeComponent();
		}

		private void rtfHelp_LinkClicked(object sender, LinkClickedEventArgs e)
		{
			Process.Start(new ProcessStartInfo {FileName = e.LinkText, UseShellExecute = true});
		}

		private void HelperForm_FormClosing(object sender, FormClosingEventArgs e)
		{
			e.Cancel = true;
			Hide();
		}

		private void rtfHelp_TextChanged(object sender, System.EventArgs e)
		{

		}
	}
}
